.github/workflows/jupyter.yaml

```
@@ -0,0 +1,58 @@
+name: "PyVAST Python Egg"
```

```suggestion
name: "Jupyter Notebook"
```

Can you add a badge to the README so we have the workflow status at a glance for master?

---

This is not yet resolved @0ortmann.